fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/StmtExecutionAction.java

```diff
@@ -100,7 +100,11 @@ public Object executeSQL(@PathVariable(value = NS_KEY) String ns, @PathVariable(
 
         ConnectContext.get().changeDefaultCatalog(ns);
         ConnectContext.get().setDatabase(getFullDbName(dbName));
-        return executeQuery(authInfo, stmtRequestBody.is_sync, stmtRequestBody.limit, stmtRequestBody);
+
+        String streamHeader = request.getHeader("X-Doris-Stream");
+        boolean isStream = !("false".equalsIgnoreCase(streamHeader));
```

Comment:
Why not use "true".equalsIgnoreCase(streamHeader), it can not use stream when streamHeader is null.

Reply:
Because I think stream should be default behavior, and can close it by setting `streamHeader: false`